src/test/javascript/spec/component/exam/manage/exam-management.component.spec.ts

```diff
@@ -84,4 +85,53 @@ describe('Exam Management Component', () => {
         // THEN
         expect(service.delete).to.have.been.calledOnce;
     });
+
```

Comment:
Could you do me a favor and add this code to the test to reset all mocks, stub etc after every test. 
```
afterEach(function () {
        sinon.restore();
    });
```